src/containers/App/App.js

```diff
@@ -9,22 +9,19 @@ class App extends Component {
   constructor(props){
     super(props)
     this.state = {
-      pageTitle: `Kanban Board`,
-
-
+      pageTitle: "Kanban Board"
     };
   };
 
   componentDidMount() {
     return this.props.loadTasks();
   };
 
-
   render() {
     return (
       <div className="App">
        <TaskListTitle title={this.state.pageTitle}/>
-       <KanbanBoard  board={this.state.kanbanBoard}/>
+       <KanbanBoard  kanbanBoard={this.state.kanbanBoard}/>
```

Comment:
There is no kanbanBoard on state in this file. I have no idea what you are passing into KanbanBoard.